doc/md/guides/overview/modern-database-ci-cd.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Modern Database CI/CD with Atlas
+id: modern-database-ci-cd
+slug: modern-database-ci-cd
+---
+
+![](https://atlasgo.io/uploads/ci-cd-guide/database-ci-cd-workflow.png)
+
+Continuous integration and continuous delivery (CI/CD) are tenets of modern software development.
+CI/CD enables teams to deliver software faster, with higher quality, and with less risk. However,
+CI/CD is not just for application code.  Teams can also apply CI/CD principles to their databases,
+to extend the benefits of CI/CD to one of the most critical components of their stack.
+
+This document describes the recommended workflow for teams using Atlas to achieve continuous integration
+and continuous delivery (CI/CD) for their databases.
+
+## Principles
+
+1. **Changes to the database are planned automatically**. Given the desired state of the database,
+   the system should automatically generate a plan for how to get from the current state to the desired
+   state.
+2. **Changes to the database schema are stored in a versioned migration directory.** All planned changes
+   to the database are checked in to a versioned migration directory.  This directory contains SQL scripts,
+   which are executed in order to apply the changes to the database.
```

Comment:
Executed in a specific order? Or executed to apply the changes? 
If the former, I'd write: which are executed in consecutive (or lexographic, if you wanna be specific) order to apply the changes to the database
If the latter, I'd write: which are executed to apply the changes to the database